nginx/Dockerfile

```diff
@@ -2,6 +2,10 @@ FROM nginx:stable-alpine
 
 RUN apk update
 
+# CVE-2024-2398
+RUN apk add --upgrade curl
```

Comment:
can do on one line :) 

RUN apk add --upgrade curl libcurl

Reply:
I tend to prefer keeping them one line each but I've made this change : )